scripts/monitor/processEvents.js

```diff
@@ -51,7 +52,7 @@ async function processNoParcelRelatedEvents(event) {
         log.info(`[${name}] Cancelling Mortgage ${_id}`)
         await Mortgage.update(
           {
-            status: Mortgage.STATUS.cancelled,
```

Comment:
While both `canceled` and `cancelled` are correct, the rest of the codebase uses `cancelled` for publication status. Can we change this one? just to be consistent